test/over_react/component_declaration/builder_integration_tests/new_boilerplate/null_safety_validate_required_props_test.dart

```diff
@@ -262,4 +271,17 @@ mixin MultipleMixinsTestPropsMixin on UiProps {
   late bool secondRequiredProp;
 }
 
+@Props(ignoreRequiredProps: {'testIgnoreOnMixin'})
```

Comment:
Could we update the tests using `MultipleMixinsTest` to indicate that ignored props are also being tested for that case? E.g., in the test descriptions or in comments within the tests.

Just some way so it's clear by looking at the tests themselves that we have test coverage for this case: ignoring some required props, and not others.